src/web3/post/commitVotes.ts

```diff
@@ -17,9 +17,10 @@ export const commitVotes = async (
     const tx = await contract.functions[
       "batchCommit((bytes32,uint256,bytes,bytes32,bytes)[])"
     ](data);
-    // console.log("commit votes TX?", tx);
+
     return tx;
   } catch (err) {
     console.log("Err in attempted commitVote transaction", err);
+    throw new Error(err.message);
```

Comment:
fyi you can just rethrow the old error: `throw err`. also do you want to keep the log here?